src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
+const resolved = document.createElement('div');
 
-// write your code here
+resolved.classList.add('message');
+resolved.textContent = 'Promise was resolved!';
+
+const rejected = document.createElement('div');
+
+rejected.classList.add('message');
+rejected.classList.add('message-error');
+rejected.textContent = 'Promise was rejected!';
+
+
+const logo = document.querySelector('.logo');
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', (e) => {
+    resolve('Success');
+    document.body.appendChild(resolved);
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('error'));
+    document.body.appendChild(rejected);
+  }, 3000);
+});
```

Comment:
Similar to the previous comment, ensure that the promise has explicit success and error handlers using `.then()` and `.catch()`.